logistics_project/apps/malawi/migrations/0016_auto__add_refrigeratormalfunction.py

```diff
@@ -247,6 +248,7 @@ def backwards(self, orm):
             'create_date': ('django.db.models.fields.DateTimeField', [], {}),
             'date': ('django.db.models.fields.DateTimeField', [], {}),
             'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
+            'is_facility': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
```

Comment:
(i actually reran the command to recreate the migration but it's good to see this is all that changed)